plugins/queue/mmap/queue_opreation.go

```diff
@@ -0,0 +1,237 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
2 things, 
1. `opreation` -> `operation`? Seems a typo
2. License issue, I crosscheck this with the following two files, they have similar codes, which I think we should remove the ASF header, because we don't own the original copyright.  Le't keep its [MIT license](https://github.com/grandecola/bigqueue/blob/master/LICENSE) in the header part and add comments about the original sources and why we need to change codes.
- https://github.com/grandecola/bigqueue/blob/master/write.go
- https://github.com/grandecola/bigqueue/blob/master/read.go

Reply:
It has been added. Please check it again